DigitalOffice/Services/CompanyService/Controllers/CompanyController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LT.DigitalOffice.CompanyService.Commands.Interfaces;
+using LT.DigitalOffice.CompanyService.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Controllers
+{[Route("api/[controller]")]
+    [ApiController]
+    public class CompanyController : ControllerBase
+    {
+        [Route("getPositionById")]
+        [HttpGet]
+        public Position GetPositionById([FromServices] IGetPositionByIdCommand command, [FromQuery] Guid positionId)
+        {
+            return command.Execute(positionId);
+        }
+    }
+   }
+
```

Comment:
Something is wrong here with characters, spaces, indentation.

Reply:
fixed